marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/matching.cy.tsx

```diff
@@ -355,10 +357,14 @@ describe("Matching", () => {
 
     // to test validation checks when user selects test among URIs only radio
     matchingStepDetail.getTestMatchUriButton();
+    cy.waitForAsyncRequest();
+    cy.wait(1000);
```

Comment:
Why are we adding some wait here? Are this test failing sometimes?

Reply:
This seem to be failing very regularly on Firefox, Since we have older versions of it, I was thinking may be waiting might be helpful. This is a trial and error situation as the tests go fine locally and it is hard to debug Firefox failure as we do not have videos as there is a bug with Firefox versions > 92. If this doesn't work. I ll remove this and disable the test for Firefox.